src/nvim/globals.h

```diff
@@ -1055,4 +1055,72 @@ typedef enum {
 // Only filled for Win32.
 EXTERN char windowsVersion[20] INIT(= { 0 });
 
+/// Command-line colors: one chunk
```

Comment:
all of these additions should be in `src/nvim/ex_getln.h` not `globals.h`

Reply:
I have tested it, but `EXTERN` does not work in `ex_getln.h`.
Please teach me the fix.

Reply:
sure. `#include "nvim/ex_getln.h"` should be added to main.c to support EXTERN/INIT.

Reply:
I get the fix.